File: RP3/Desafio2.cpp
#include <iostream>
using namespace std;

int main() {
    int numero = 0;

    while (numero <= 10) {
        cout << "Digite um número: ";
        cin >> numero;
    }

    cout << "Obrigado! Você digitou um número maior que 10." << endl;

    return 0;
}

File: RP3/Desafio2.java
import java.util.Scanner;

public class Desafio2 {
    public static void main(String[] args) {

        try (Scanner sc = new Scanner(System.in)) {

            int numero = 0;

            while (numero <= 10) {
                System.out.print("Digite um número: ");
                numero = sc.nextInt();
            }

            System.out.println("Obrigado! Você digitou um número maior que 10.");
        }
    }
}